app/views/sol_prices/index.html.erb

```diff
@@ -8,9 +8,6 @@
       <a class="btn btn-sm btn-secondary me-1 solPricesChartTabButton <%= active_button?(:coin_gecko) %>" href="#coinGeckoChartTab">
```

Comment:
I'd comment this out since there's no other choice

Reply:
Do we anticipate adding new sol price client in a short time? I don't think so, so I removed this code as well as logic within the sol price component.

Reply:
Let me know if you agree with my above action.